src/box/alter.cc

```diff
@@ -4348,6 +4349,10 @@ on_replace_dd_schema(struct trigger * /* trigger */, void *event)
 				return -1;
 		}
 		data->fiber = fiber;
+		/* Persist consumers on upgrade to 3.2.0 */
+		if (version == version_id(3, 2, 0) &&
+		    gc_persist_consumers() != 0)
+			return -1;
```

Comment:
Why do it in the same transaction where we update the schema version? Why isn't is okay to persist consumers asynchronously?